transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringEventLoop.java

```diff
@@ -220,9 +223,8 @@ void handleLoopException(Throwable t) {
     @Override
     public void handle(int fd, int res, int flags, int op, int data) {
         if (op == Native.IORING_OP_READ && eventfd.intValue() == fd) {
-            if (res != Native.ERRNO_ECANCELED_NEGATIVE) {
-                addEventFdRead(ringBuffer.ioUringSubmissionQueue());
-            }
+            pendingWakeup = false;
+            addEventFdRead(ringBuffer.ioUringSubmissionQueue());
```

Comment:
@njhill shouldn't we keep the `cancel` check ?

Reply:
I don't think it can happen since we never cancel the blocking read. I think it was left over from when we were using non-blocking with poll.